code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -0,0 +1,235 @@
+
+	var/datum/species/X = mob_species
+	var/golem_forename = initial(X.id)
+
+	// The id of golem species is either their material "diamond","gold",
+	// or just "golem" for the plain ones. So we're using it for naming.
+
+	if(golem_forename == "golem")
+		golem_forename = "iron"
+
+	new_spawn.real_name = "[capitalize(golem_forename)] [golem_surname]"
+	// This means golems have names like Iron Forge, or Diamond Quarry
+	// also a tiny chance of being called "Plasma Meme"
+	// which is clearly a feature
+
+	new_spawn << "Build golem shells in the autolathe, and feed refined mineral sheets to the shells to bring them to life! You are generally a peaceful group unless provoked."
+	if(ishuman(new_spawn))
+		var/mob/living/carbon/human/H = new_spawn
+		H.set_cloned_appearance()
+
+
+/obj/effect/mob_spawn/human/golem/adamantine
+	name = "dust-caked golem shell"
+	desc = "A humanoid shape, empty, lifeless, and full of potential."
+	mob_name = "a free golem"
+	anchored = 1
+	density = 1
+	mob_species = /datum/species/golem/adamantine
+
+//Malfunctioning cryostasis sleepers: Spawns in makeshift shelters in lavaland. Ghosts become hermits with knowledge of how they got to where they are now.
+/obj/effect/mob_spawn/human/hermit
+	name = "malfunctiong cryostasis sleeper"
+	desc = "A humming sleeper with a silhoutted occupant inside. Its stasis function is broken and it's likely being used as a bed."
+	mob_name = "a stranded hermit"
+	icon = 'icons/obj/lavaland/spawners.dmi'
+	icon_state = "cryostasis_sleeper"
+	roundstart = FALSE
+	death = FALSE
+	random = TRUE
+	mob_species = /datum/species/human
+	flavour_text = "<font size=3><b>Y</b></font><b>ou've been stranded in this godless prison of a planet for longer than you can remember. Each day you barely scrape by, and between the terrible \
+	conditions of your makeshift shelter, the hostile creatures, and the ash drakes swooping down from the cloudless skies, all you can wish for is the feel of soft grass between your toes and \
+	the fresh air of Earth. These thoughts are dispelled by yet another recollection of how you got here... "
+
+/obj/effect/mob_spawn/human/hermit/New()
+	var/arrpee = rand(1,4)
+	switch(arrpee)
+		if(1)
+			flavour_text += "you were a [pick("arms dealer", "shipwright", "docking manager")]'s assistant on a small trading station several sectors from here. Raiders attacked, and there was \
+			only one pod left when you got to the escape bay. You took it and launched it alone, and the crowd of terrified faces crowding at the airlock door as your pod's engines burst to \
+			life and sent you to this hell are forever branded into your memory.</b>"
+			uniform = /obj/item/clothing/under/assistantformal
+			shoes = /obj/item/clothing/shoes/sneakers/black
+			back = /obj/item/weapon/storage/backpack
+		if(2)
+			flavour_text += "you're an exile from the Tiger Cooperative. Their technological fanaticism drove you to question the power and beliefs of the Exolitics, and they saw you as a \
```

Comment:
This is all fine except for the shit about Exolitics, what the fuck is an Exolitic it's not in the code anywhere